### PR TITLE
Fix EN translation from about.html template

### PR DESCRIPTION
**Pull request by ibianka**

### CODE REVIEW

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -20,52 +20,22 @@ msgstr ""
...
-#: AlertaDengue/common/templates/upload_successful.html:5
-#: AlertaDengue/dbf/templates/upload_successful.html:5
-msgid "Dados do SINAN recebidos"
-msgstr "SINAN data received"
-
 #: AlertaDengue/dados/charts/cities.py:74
-#, fuzzy
-#| msgid "Limiar epidêmico"
 msgid "limiar pré epidêmico"
 msgstr "Epidemic threshold"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:25]: should be "pre-epidemic threshold"

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -20,52 +20,22 @@ msgstr ""
...
 
 #: AlertaDengue/dados/charts/cities.py:75
-#, fuzzy
-#| msgid "Limiar epidêmico"
 msgid "limiar pós epidêmico"
 msgstr "Epidemic threshold"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:29]: should be "Post-epidemic threshold"

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -123,26 +93,18 @@ msgid "Registrados"
...
-#, fuzzy, python-brace-format
-#| msgid "%{x} <br>Semana: %{text} <br>%{y} Casos Estimados<extra></extra>"
 msgid "<br>SE %{text}<br>%{y:1f} Casos estimados<extra></extra>"
 msgstr "%{x} <br>Week: %{text} <br>%{y} Estimated Cases<extra></extra>"
 
 #: AlertaDengue/dados/charts/home.py:75
-#, fuzzy
-#| msgid "Número acumulado de casos no ano"
 msgid "Total de casos no estado"
 msgstr "Accumulated number of cases per year"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:109]: It seems that this refers to the "cumulative number of cases in the state"

**Review by fccoelho — CHANGES_REQUESTED** [review]
I would be nice to also check if the base text still corresponds to the actual content of the page, as sometimes the content changes but the HTML does not get updated.

File: AlertaDengue/locale/en/LC_MESSAGES/django.po (hunk 1 above)
Comment (esloch) [reply AlertaDengue/locale/en/LC_MESSAGES/django.po:25]: The template [reports by cities] does not seem to be translated yet on the site;(https://github.com/AlertaDengue/AlertaDengue/blob/main/AlertaDengue/dados/templates/report_city.html#L95)

Just the [lines from the charts] module that @ibianka has fuzzy removed.
https://github.com/AlertaDengue/AlertaDengue/blob/main/AlertaDengue/locale/en/LC_MESSAGES/django.po#L45

File: AlertaDengue/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (esloch) [reply AlertaDengue/locale/en/LC_MESSAGES/django.po:29]: Here, we don't need to translate list items for iteration (lines 74,75,76).

``` 
        ks_limiar = [
            _('limiar pré epidêmico'),
            _('limiar pós epidêmico'),
            _('limiar epidêmico'),
        ]

        colors = ['rgb(0,255,0)', 'rgb(255,150,0)', 'rgb(255,0,0)']

        for k, c in zip(ks_limiar, colors):
            figure.add_trace(
                go.Scatter(
                    x=df['SE'],
                    y=df[k],
                    y0=df['incidência'],
                    name=k.title(),
                    marker={'color': c},
                    text=df.SE.map(lambda v: '{}'.format(str(v)[-2:])),
                    hoverinfo='text',
                    hovertext=df['incidência'],
                    hovertemplate="Semana %{text} <br>Incidência=%{y:1f}",
                ),
                secondary_y=False,
            )
```

File: AlertaDengue/locale/en/LC_MESSAGES/django.po (hunk 3 above)
Comment (esloch) [reply AlertaDengue/locale/en/LC_MESSAGES/django.po:109]: I recommend doing the translations for each page in different pull requests.

**Comment by ibianka** [comment]
Thanks, @fccoelho, and @esloch  for the review!
I fix the list from cities.py and change the message string from home.py
I will translate each template in different pull request

**Pull request merged**